ern-container-gen/src/generators/android/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeMiniAppActivity.java

```diff
@@ -111,4 +117,25 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
     public void onBackKey() {
         finish();
     }
+
+    @Override
+    public int checkPermission(String permission, int pid, int uid) {
+        return PackageManager.PERMISSION_GRANTED;
```

Comment:
This method is supposed to return `PERMISSION_GRANTED` or `PERMISSION_DENIED` based on if the permission is allowed for a particular pid/uid. Overriding with permission granted may have an inverse effect for the feature ?.